content/influxdb/v2.1/notebooks/overview.md

```diff
@@ -10,72 +10,67 @@ menu:
     parent: Notebooks
 ---
 
-Learn about the building blocks of a notebook, how notebooks can help show how data is processed, and discover some common use cases.
+Learn how notebooks can help to streamline and simplify your day-to-day business processes.
 
-To use different notebook cell types, controls, and uses to process data, see:
-- [Notebook concepts](/influxdb/v2.1/notebooks/overview/#notebook-concepts)
-- [Notebook controls](/influxdb/v2.1/notebooks/overview/#notebook-controls)
-- [Notebook cell types](/influxdb/v2.1/notebooks/overview/#notebook-cell-types)
+See an overview of [notebook concepts](/influxdb/v2.1/notebooks/overview/#notebook-concepts), [notebook controls](/influxdb/v2.1/notebooks/overview/#notebook-controls), and [notebook cell types](/influxdb/v2.1/notebooks/overview/#notebook-cell-types) also know as the basic building blocks of a notebook.
 
 ## Notebook concepts
 
-Using notebooks, you can:
+You can think of an InfluxDB notebook as template that contains cells (building blocks) to define how to query, visualize, process, and write data to your buckets. Notebooks help you do the following:
 
-- Create documents with live code, equations, visualizations, and explanatory notes.
-- Create dashboard cells or scheduled tasks.
-- Clean and downsample data.
-- Build runbooks.
-- Document data processing steps.
+- Create snippets of live code, equations, visualizations, and explanatory notes.
+- Create alerts or scheduled tasks.
+- Downsample and normalize data.
+- Build runbooks to share with your teams.
+- Output data to buckets.
 
 ## Notebook controls
 
 The following options appear at the top of each notebook.
 
-### Preview/Run mode
+### Run
 
-- Select **Preview** (or press **Control+Enter**) to display results of each cell without writing data. Helps to verify that cells return expected results before writing data.
-- Select **Run** mode displays results of each cell and writes data to the selected bucket.
+Select {{< caps >}}Run{{< /caps >}} (or press **Control+Enter**) to display results of each cell and write data to the selected bucket.
 
-### Presentation mode
+### Save Notebook
 
-Display notebooks in full screen with Presentation mode, hiding the left and top navigation menus so only the cells appear. This mode might be helpful, for example, for stationary screens dedicated to monitoring visualizations.
+Select {{< caps >}}Save Notebook{{< /caps >}} to save all notebook cells.
+
+{{% note %}}
+Saving the notebook does not save cell results. When you open a saved notebook, click {{< caps >}}**Run**{{< /caps >}} to update cell results. {{% /note %}}
```

Comment:
- Did you purposefully remove Presentation mode? This is still a thing.
- I'm not seeing "Save Notebook" in either OSS or Cloud. It appears they now both autosave whenever notebooks are modified.

Reply:
no, deleted Presentation by accident
 
Will note that his is only an option before the first-time save...does this feel like unusual behavior that the first time you need to manually save, but thereafter it auto resaves? cc @barbaranelson